js/index.js

```diff
@@ -415,8 +415,8 @@
 
                 var geoJSON = L.geoJson(turf.featureCollection(cleanedGeoJSONFeatures), {
                     onEachFeature: function (feature, layer) {
-                        if (!feature.properties.nogoWeight) {
-                            feature.properties.nogoWeight = nogoWeight;
+                        if (!feature.properties.nogoWeight && nogoWeight != -1) {
+                            layer.options.nogoWeight = feature.properties.nogoWeight || nogoWeight;
```

Comment:
This doesn't cover yet the case when `feature.properties.nogoWeight` is set (as it does *not* automatically get copied over, as probably previously assumed here)

Reply:
Indeed, sorry… Thanks for the review! This should be fixed.